jetson/filter/src/__main__.py

```diff
@@ -404,6 +476,13 @@ async def run(self):
 
                 odom = self.state_estimate.asOdom()
                 self.lcm.publish('/odometry', odom.encode())
+
+                corrected_bearing = bearing + self.bearing_offset
+                corrected_bearing_struct = CorrectedBearing()
```

Comment:
i guess to save some space you could use the same Odometry object but publish the corrected_bearing to this one's bearing and put it to "/corrected_bearing", gives extra info but i think for testing it's an easy add on so we don't need to delete the CorrectedBearing lcm after we're done testing/leave a random unused one hanging

Reply:
would put extra unnecessary overhead onto the network tho but i dont think it's a big deal for testing

Reply:
I mean either way we still have to remember to delete something after testing it, I think its gonna be easier to just leave it like this and ill be sure to get rid of that LCM struct once were done testing this